### PR TITLE
Allow adding custom labels and annotations to the pod created by PodRunner

### PR DESCRIPTION
**Pull request by pavannd1**
## Change Overview

- Add labels and annotations to PodOptions
- Unit test
- Delete/rename/replace biased words 🚀 

Review commit wise

## Pull request type

Please check the type of change your PR introduces:

- [x] :hamster: Trivial/Minor
- [x] :sunflower: Feature

## Test Plan

- [x] :muscle: Manual
- [x] :zap: Unit test

### CODE REVIEW

File: pkg/kube/pod.go [hunk 1]
```diff
@@ -105,6 +107,15 @@ func CreatePod(ctx context.Context, cli kubernetes.Interface, opts *PodOptions)
...
 		Spec: patchedSpecs,
 	}
+
+	// Add Annotations and Labels, if specified
+	if opts.Annotations != nil {
+		pod.ObjectMeta.Annotations = opts.Annotations
+	}
+	if opts.Labels != nil {
+		pod.ObjectMeta.Labels = opts.Labels
+	}
```
Comment (onkarbhat) [inline pkg/kube/pod.go:117]: Is there any form of validation required for the labels and annotations ? Or is it fine to copy everything ?

File: examples/stable/couchbase/README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ $ kubectl create -f couchbase-blueprint.yaml -n kasten-io
 blueprint.cr.kanister.io/couchbase-blueprint created
 ```
 
-Once Couchbase is running, you can populate it with some data. Let's add some dummy documents to the default bucket:
+Once Couchbase is running, you can populate it with some data. Let's add some documents to the default bucket:
```
Comment (onkarbhat) [inline examples/stable/couchbase/README.md:75]: 💯

File: pkg/kube/volume/volume.go [hunk 3]
```diff
@@ -253,15 +253,15 @@ func DeletePVC(cli kubernetes.Interface, namespace, pvcName string) error {
 	})
 }
 
-var labelBlackList = map[string]struct{}{
+var labelDenyList = map[string]struct{}{
```
Comment (onkarbhat) [inline pkg/kube/volume/volume.go:256]: 👍

File: pkg/kube/pod.go (hunk 1 above)
Comment (pavannd1) [reply pkg/kube/pod.go:117]: Not really required since they are just a `map[string]string` and could have anything as key/value pairs

**Pull request merged**